src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/StepSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { Tooltip } from 'antd'
+import { ThemeContext } from 'contexts/themeContext'
+import { PropsWithChildren, useContext } from 'react'
+import { CheckCircleFilled } from '@ant-design/icons'
+import { Trans } from '@lingui/macro'
+
+export function StepSection({
```

Comment:
Compone used in each section of the set up modal.

Reply:
<img width="431" alt="Screen Shot 2022-07-02 at 7 33 35 PM" src="https://user-images.githubusercontent.com/94939382/176995925-0f88ccb7-7071-4065-bda2-860137d24735.png">